lib/Migrator/APIDiffMigratorPass.cpp

```diff
@@ -695,9 +695,10 @@ struct APIDiffMigratorPass : public ASTMigratorPass, public SourceEntityWalker {
           if (ArgLoc.isInvalid())
             Editor.insertBefore(PD->getNameLoc(),
                                 (llvm::Twine(NewArg) + " ").str());
```

Comment:
Should there be a ternary checking NewArg is empty here too?

Reply:
Good catch, we've not seen this kind of change though. Will add here too.